swap_meet/item.py

```diff
@@ -1,2 +1,30 @@
 class Item:
-    pass
+    def __init__(self, category = "", condition = 0.0,):
```

Comment:
Watch out for the stray trailing comma here. I _think_ it's technically ok, in the same way that we can have an extra trailing comma in a list or dictionary literal, but for some reason it makes me feel really uncomfortable here.